charts/node/templates/service.yaml

```diff
@@ -29,6 +29,10 @@ spec:
     - port: {{ .Values.node.prometheus.port | int }}
       name: prometheus
     {{- end }}
+    {{- if $.Values.node.collator.relayChainPrometheus.enabled }}
+    - port: {{ $.Values.node.perNodeServices.apiService.relayChainPrometheusPort}}
```

Comment:
add `| int` ?

Reply:
Added it for all other ports in the file